caas/application.go

```diff
@@ -155,10 +155,10 @@ type ContainerConfig struct {
 	Mounts []MountConfig
 
 	// Uid to run as. Default to 0 or root.
```

Comment:
Not sure the comment is still accurate. Isn't the default now "unset".
Same below.